doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,105 @@
+"""
+===================================================
+Explore and visualize region properties with pandas
+===================================================
+
+This toy example shows how to compute the size of every labelled region in a
+series of 10 images. We use 2D images and then 3D images. The blob-like
+regions are generated synthetically. As the volume fraction (i.e., number of
+pixels or voxels covered by the blobs) increases, the number of blobs
+(regions) decreases, and the size (area or volume) of a single region can get
+larger and larger. The area (size) values are available in a pandas-compatible
+format, which makes for convenient data analysis and visualization.
+
+Beyond area, many other region properties are available.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from skimage import data, measure
+
+
+fractions = np.linspace(0.05, 0.5, 10)
+
+"""
+2D images
+---------
+"""
+
+images = [data.binary_blobs(volume_fraction=f) for f in fractions]
+
+labeled_images = [measure.label(image) for image in images]
+
+properties = ['label', 'area']
+
+tables = [measure.regionprops_table(image, properties=properties)
+          for image in labeled_images]
+tables = [pd.DataFrame(table) for table in tables]
+
+for fraction, table in zip(fractions, tables):
+    table['volume fraction'] = fraction
+
+areas = pd.concat(tables, axis=0)
+
+# Divide the figure into a 2x1 grid
+fig, (im, ax) = plt.subplots(2, 1)
+# Turn off axes for top subplot
+im.axis('off')
+# Plot area vs volume fraction
+areas.plot(x='volume fraction', y='area', kind='scatter', ax=ax)
+# Show image with lowest volume fraction
+ax1 = fig.add_subplot(221)
+ax1.imshow(images[0])
+# Show image with highest volume fraction
+ax2 = fig.add_subplot(222)
+ax2.imshow(images[-1])
+plt.show()
+
+"""
+In the scatterplot, many points seem to be overlapping at low area values.
+To get a better sense of the distribution, we may want to add some 'jitter'
+to the visualization. To this end, we use `stripplot` (from `seaborn`, the
+Python library dedicated to statistical data visualization) with argument
+`jitter=True`.
+"""
+
+sns_fig, sns_ax = plt.subplots()
+sns.stripplot(x='volume fraction', y='area', data=areas, jitter=True,
+              ax=sns_ax)
+x_format = sns_ax.xaxis.get_major_formatter()
+x_format.seq = ['{:0.2f}'.format(float(s)) for s in x_format.seq]
+sns_ax.xaxis.set_major_formatter(x_format)
+plt.show()
+
+"""
+3D images
+---------
+"""
```

Comment:
I think this syntax no longer works in sphinx-gallery, and this needs to be a comment (see other examples), but @emmanuelle knows best... Personally I like this better, if it works!

Finally, I would add a paragraph to 3D: "Doing the same analysis in 3D, we find a completely different behavior, with blobs coalescing into a single, giant component as the volume fraction crosses ~0.25."

Otherwise, I really like how you've incorporated both pandas and seaborn plotting into the example! 🎉

Reply:
> I think this syntax no longer works in sphinx-gallery, and this needs to be a comment (see other examples), but @emmanuelle knows best... Personally I like this better, if it works!

Ok, I have found Markdown in between code at https://scikit-image.org/docs/dev/auto_examples/developers/plot_max_tree.html; will use this comment-like syntax then.

Reply:
Done in c747775.

> "Doing the same analysis in 3D, we find a completely different behavior

It's not obvious to me (judging only from this example) that it would be _qualitatively_ (and, hence, completely) different... Some kind of phase transition would need to be evidenced for me to call it a "completely different behaviour." So I've described it as "much more dramatic," to stay of the safe side.